docs/qaoa/qaoa_overview.rst

```diff
@@ -8,7 +8,7 @@ Structure
 _________
 
 The pyQAOA package contains separate modules for each type of problem
-instance: MAX-CUT, K-SAT, etc.
+instance: MAX-CUT, graph partitioning, etc.
 Each problem instance is a class that inherits the main QAOA object and
```

Comment:
This is no longer how the code is structured.  No override is necessary. 

maybe something like...."For each problem instance the user specifics the driver Hamiltonian, cost Hamiltonian, and the approximation order of the algorithm."